packages/oui-header-tabs/src/header-tab.directive.js

```diff
@@ -0,0 +1,32 @@
+import _ from "lodash";
+import controller from "./header-tab.controller";
+import template from "./header-tab.html";
+
+export default () => ({
+    template,
+    controller,
+    replace: true,
+    restrict: "E",
+    require: ["^^ouiHeaderTabs", "ouiHeaderTab"],
+    controllerAs: "$ctrl",
+    bindToController: true,
+    scope: {
+        state: "@",
```

Comment:
No `href` attribute ? Not all managers use states. 
The tabs components must be thought for a general use.